testscripts/encode.py

```diff
@@ -0,0 +1,79 @@
+import os
+import sys
+import time
+import shutil
+from yamilib import libyami
+
+def encode(yamidecode, yamiencode, psnr, testfile, output):
+    outputname = output + os.sep + os.path.basename(testfile) + '.h264'
+    decodename = output + os.sep + os.path.basename(testfile)
+    DecodeObject = libyami(yamidecode, yamiencode, psnr, testfile, outputname)
+    DecodeObject.yamiencodefunc(DecodeObject.testfile, DecodeObject.outputname)
+    DecodeObject.yamidecodefunc(DecodeObject.outputname, decodename)
+    W_H = DecodeObject.get_w_h(testfile)
+    DecodeObject.psnrfunc(testfile, decodename, W_H, DecodeObject.ENCODE_PSNR)
+    os.remove(outputname)
+    os.remove(decodename)
+    os.remove(testfile)
+    
+if __name__ == '__main__':
```

Comment:
compare this to decode.py, seems only diff is encode and decode function, is it possible to merge other code?